packages/babel-preset-env/test/fixtures/preset-options/destructuring-edge/output.js

```diff
@@ -1,8 +1,8 @@
 ((a, {
   b: _b = 0,
-  c: _c = 3
+  c = 3
```

Comment:
This code should be transformed, because Edge 15 doesn't support default values in destructuring in parameters of arrow functions.

I don't understand why the first property is still transformed, but the second one isn't 🤔 

The source code of this transform is https://github.com/babel/preset-modules/blob/master/src/plugins/transform-edge-default-parameters/index.js

Reply:
Looks like the reason is setting:

```js
path.parent.shorthand = false;
(path.parent.extra || {}).shorthand = false;
```

And if `shorthand` set to `false` no fix for `ObjectProperty` occurred.

Reply:
Also first version of my PR didn't affect additional fixtures, should I revert to previous version?

@liuxingbaoyu have you test this case locally? My tests with "additional traversing" worked good.